src/read.jl

```diff
@@ -114,6 +115,7 @@ function read_demand_flexibility(filepath, interval)::DemandFlexibility
         "enabled" => "not_specified",
         "interval_balance" => true,
         "rolling_balance" => true,
+        "source" => "doe",
```

Comment:
It seems like this `source` parameter controls a couple of things if it's set to `"doe"`:
- the filepath that flexibility information is loaded from
- that bus flexibility needs to come from mapping each bus's EIA ID to the corresponding column in the loaded CSV
- that flexibility is interpreted as a percentage of demand, not absolute MW
- others?

Are there situations where we may want to be doing some of these things, but not others? If so, we might want to make this control a bit more granular.

Reply:
1. The filepath is a temporary thing for now because ideally this file will be on the blob server.
2. yes, it is assumed the case file have an EIA for each bus, which will be done outside REISE
3. right 
now it is either from user specified input csvs (like before) or from DOE data so it only does one of the 2 things. If it is to be generalized there would need a few more parameters.